hack/tools/release/weekly/main.go

```diff
@@ -50,47 +53,63 @@ var (
 		release.Unknown,
 	}
 
-	from = flag.String("from", "", "Include commits starting from and including this date. Accepts format: YYYY-MM-DD")
-	to   = flag.String("to", "", "Include commits up to and including this date. Accepts format: YYYY-MM-DD")
+	since   string
+	until   string
+	branch  string
+	ghToken string
 
-	milestone = flag.String("milestone", "v1.4", "Milestone. Accepts format: v1.4")
+	timeLayout = "2006-01-02"
 
 	tagRegex = regexp.MustCompile(`^\[release-[\w-\.]*\]`)
 )
 
 func main() {
+	flag.StringVar(&since, "since", "", "Include commits starting from and including this date. Accepts format: YYYY-MM-DD")
+	flag.StringVar(&until, "until", "", "Include commits up to and including this date. Accepts format: YYYY-MM-DD")
+	flag.StringVar(&branch, "branch", "release-1.6", "Release branch. Accepts formats: main, release-1.6")
+	flag.StringVar(&ghToken, "gh-token", "", "Github token. Enables a higher api request rate.")
```

Comment:
Instead of using a flag, I think it would be better to get the GitHub token from the `GITHUB_TOKEN` environment variable, which is used in other places for the release process.